server/api/study.py

```diff
@@ -255,58 +255,45 @@ def _requireStudyCreator(self, user):
 
     @describeRoute(
         Description('Create an annotation study.')
-        .param('name', 'The name of the study.', required=True)
-        .param('featuresetId', 'The featureset ID of the study.', required=True)
+        .param('name', 'The name of the study.')
+        .param('featuresetId', 'The featureset ID of the study.')
+        .param('userIds',
+               'The annotators user IDs of the study, as a JSON array.')
+        .param('segmentationIds',
+               'The segmentation IDs of the study, as a JSON array.')
         .errorResponse('Write access was denied on the parent folder.', 403)
     )
     @access.user
     def createStudy(self, params):
-        # Support using both query parameters and arguments as JSON (legacy)
-        body_json = None
-        try:
-            body_json = self.getBodyJson()
-        except RestException:
-            pass
-
-        study = None
-
-        if body_json is not None:
-            # Use JSON
-            self.requireParams(('name', 'annotatorIds', 'segmentationIds', 'featuresetId'), body_json)
-
-            name = body_json['name']
-            user = self.getCurrentUser()
-            self._requireStudyCreator(user)
-            annotator_users = [
-                self.model('user').load(
-                    annotator_id, user=user, level=AccessType.READ)
-                for annotator_id in body_json['annotatorIds']
-            ]
-            segmentations = [
-                self.model('segmentation', 'isic_archive').load(segmentation_id)
-                for segmentation_id in body_json['segmentationIds']
-            ]
-            featureset = self.model('featureset', 'isic_archive').load(body_json['featuresetId'])
-            if not featureset:
-                raise ValidationException('Invalid featureset ID.', 'featuresetId')
-
-            study = self.model('study', 'isic_archive').createStudy(
-                name, user, featureset, annotator_users, segmentations)
-        else:
-            # Use query parameters
-            self.requireParams(('name', 'featuresetId'), params)
-            user = self.getCurrentUser()
-            self._requireStudyCreator(user)
-
-            name = params['name'].strip()
+        if cherrypy.request.headers['Content-Type'] == 'application/json':
```

Comment:
The reason I didn't check the header like this is that calling this endpoint through Swagger sends `Content-Type:application/json`, and the response is then:

    {
      "message": "Invalid JSON passed in request body.",
      "type": "rest"
    }